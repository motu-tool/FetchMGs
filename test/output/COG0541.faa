version https://git-lfs.github.com/spec/v1
oid sha256:f614b38c9c1adbbeb624d2400ccf7feb602cb584d2a564ce663b745e862b1493
size 3099
