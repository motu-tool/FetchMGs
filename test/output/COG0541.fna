version https://git-lfs.github.com/spec/v1
oid sha256:aa265601c4829b415bf738b0c09be7c66445308f5020b2fe1735035af0d0e6a9
size 7822
