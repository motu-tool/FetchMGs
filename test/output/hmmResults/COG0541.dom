#                                                                            --- full sequence --- -------------- this domain -------------   hmm coord   ali coord   env coord
# target name        accession   tlen query name           accession   qlen   E-value  score  bias   #  of  c-Evalue  i-Evalue  score  bias  from    to  from    to  from    to  acc description of target
#------------------- ---------- ----- -------------------- ---------- ----- --------- ------ ----- --- --- --------- --------- ------ ----- ----- ----- ----- ----- ----- ----- ---- ---------------------
1089456.NCGM2_4879   -            457 COG0541              -            731  2.7e-231  770.6  27.7   1   1  6.5e-235    3e-231  770.5  27.7   176   616     1   450     1   457 0.94 protein_id="BAK91702.1" gene="ffh" product="signal recognition particle protein" GI="348036342" note="similar to GI:116051744; similar to PA14_15960 of P. aeruginosa UCBPP-PA14" project_id="73453"
1037409.BJ6T_04500   -            515 COG0541              -            731  1.2e-218  728.8  25.7   1   1  3.2e-222  1.5e-218  728.5  25.7   176   635     1   468     1   480 0.93 protein_id="BAL05748.1" product="signal recognition particle protein" GI="354953069" project_id="67463"
1148.SYNGTS_1731     -            482 COG0541              -            731  6.1e-215  716.6  20.3   1   1  1.6e-218  7.2e-215  716.3  20.3   176   645     1   476     1   481 0.93 protein_id="BAK50479.1" gene="ffh" product="signal recognition particle protein" GI="339273992" note="ORF_ID:slr1531" project_id="67081"
944547.ABLL_2406     -            450 COG0541              -            731  4.3e-174  581.5  15.8   1   1    1e-177  4.8e-174  581.4  15.8   176   620     1   445     1   450 0.94 protein_id="BAK74281.1" product="signal recognition particle protein" GI="345472831" project_id="61721"
944546.ABED_1783     -            450 COG0541              -            731  1.5e-171  573.1  17.1   1   1  3.7e-175  1.7e-171  572.9  17.1   176   620     1   445     1   450 0.94 protein_id="BAK71500.1" product="signal recognition particle protein" GI="345470049" project_id="61719"
#
# Program:         hmmsearch
# Version:         3.3.2 (Nov 2020)
# Pipeline mode:   SEARCH
# Query file:      /nfs/home/fieldc/.local/lib/python3.7/site-packages/fetchmgs/data/COG0541.hmm
# Target file:     ../fetchMGs/example_datasets/example_data_genomes.faa
# Option settings: /nfs/home/fieldc/.local/lib/python3.7/site-packages/fetchmgs/bin/hmmsearch -o output/hmmResults/COG0541.out --domtblout output/hmmResults/COG0541.dom --noali --notextw -T 450 --cpu 1 /nfs/home/fieldc/.local/lib/python3.7/site-packages/fetchmgs/data/COG0541.hmm ../fetchMGs/example_datasets/example_data_genomes.faa 
# Current dir:     /nfs/nas22/fs2202/biol_micro_bioinf/software/fetchmgs_dev
# Date:            Wed Sep 14 14:19:50 2022
# [ok]
