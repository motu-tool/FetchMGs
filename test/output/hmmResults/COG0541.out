# hmmsearch :: search profile(s) against a sequence database
# HMMER 3.3.2 (Nov 2020); http://hmmer.org/
# Copyright (C) 2020 Howard Hughes Medical Institute.
# Freely distributed under the BSD open source license.
# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
# query HMM file:                  /nfs/home/fieldc/.local/lib/python3.7/site-packages/fetchmgs/data/COG0541.hmm
# target sequence database:        ../fetchMGs/example_datasets/example_data_genomes.faa
# output directed to file:         output/hmmResults/COG0541.out
# per-dom hits tabular output:     output/hmmResults/COG0541.dom
# show alignments in output:       no
# max ASCII text line length:      unlimited
# sequence reporting threshold:    score >= 450
# number of worker threads:        1
# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

Query:       COG0541  [M=731]
Scores for complete sequences (score includes all domains):
   --- full sequence ---   --- best 1 domain ---    -#dom-
    E-value  score  bias    E-value  score  bias    exp  N  Sequence           Description
    ------- ------ -----    ------- ------ -----   ---- --  --------           -----------
   2.7e-231  770.6  27.7     3e-231  770.5  27.7    1.0  1  1089456.NCGM2_4879  protein_id="BAK91702.1" gene="ffh" product="signal recognition particle protein" GI="348036342" note="similar to GI:116051744; similar to PA14_15960 of P. aeruginosa UCBPP-PA14" project_id="73453"
   1.2e-218  728.8  25.7   1.5e-218  728.5  25.7    1.0  1  1037409.BJ6T_04500  protein_id="BAL05748.1" product="signal recognition particle protein" GI="354953069" project_id="67463"
   6.1e-215  716.6  20.3   7.2e-215  716.3  20.3    1.0  1  1148.SYNGTS_1731    protein_id="BAK50479.1" gene="ffh" product="signal recognition particle protein" GI="339273992" note="ORF_ID:slr1531" project_id="67081"
   4.3e-174  581.5  15.8   4.8e-174  581.4  15.8    1.0  1  944547.ABLL_2406    protein_id="BAK74281.1" product="signal recognition particle protein" GI="345472831" project_id="61721"
   1.5e-171  573.1  17.1   1.7e-171  572.9  17.1    1.0  1  944546.ABED_1783    protein_id="BAK71500.1" product="signal recognition particle protein" GI="345470049" project_id="61719"


Domain annotation for each sequence:
>> 1089456.NCGM2_4879  protein_id="BAK91702.1" gene="ffh" product="signal recognition particle protein" GI="348036342" note="similar to GI:116051744; similar to PA14_15960 of P. aeruginosa UCBPP-PA14" project_id="73453"
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !  770.5  27.7  6.5e-235    3e-231     176     616 ..       1     450 [.       1     457 [] 0.94

>> 1037409.BJ6T_04500  protein_id="BAL05748.1" product="signal recognition particle protein" GI="354953069" project_id="67463"
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !  728.5  25.7  3.2e-222  1.5e-218     176     635 ..       1     468 [.       1     480 [. 0.93

>> 1148.SYNGTS_1731  protein_id="BAK50479.1" gene="ffh" product="signal recognition particle protein" GI="339273992" note="ORF_ID:slr1531" project_id="67081"
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !  716.3  20.3  1.6e-218  7.2e-215     176     645 ..       1     476 [.       1     481 [. 0.93

>> 944547.ABLL_2406  protein_id="BAK74281.1" product="signal recognition particle protein" GI="345472831" project_id="61721"
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !  581.4  15.8    1e-177  4.8e-174     176     620 ..       1     445 [.       1     450 [] 0.94

>> 944546.ABED_1783  protein_id="BAK71500.1" product="signal recognition particle protein" GI="345470049" project_id="61719"
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !  572.9  17.1  3.7e-175  1.7e-171     176     620 ..       1     445 [.       1     450 [] 0.94



Internal pipeline statistics summary:
-------------------------------------
Query model(s):                            1  (731 nodes)
Target sequences:                      23272  (7230323 residues searched)
Passed MSV filter:                      1942  (0.0834479); expected 465.4 (0.02)
Passed bias filter:                     1154  (0.0495875); expected 465.4 (0.02)
Passed Vit filter:                       302  (0.012977); expected 23.3 (0.001)
Passed Fwd filter:                        43  (0.00184771); expected 0.2 (1e-05)
Initial search space (Z):              23272  [actual number of targets]
Domain search space  (domZ):               5  [number of targets reported over threshold]
# CPU time: 0.68u 0.01s 00:00:00.69 Elapsed: 00:00:00.58
# Mc/sec: 9036.75
//
[ok]
